Stratis.Bitcoin.Features.Consensus/ConsensusLoop.cs

```diff
@@ -215,6 +427,11 @@ public void AcceptBlock(ContextInformation context)
             this.logger.LogTrace("(-)");
         }
 
+        /// <summary>
+        /// This method try to load from cash the items of the next block, in a background task. 
```

Comment:
cach`e`

Reply:
and maybe not items, but UTXOs ? or what do you mean by items?

Reply:
fixed